addons/bus/static/tests/bus_logger.test.js

```diff
@@ -0,0 +1,17 @@
+import { Logger } from "@bus/workers/websocket_worker_utils";
+
+import { describe, expect, test } from "@odoo/hoot";
+import { advanceTime } from "@odoo/hoot-dom";
+
+describe.current.tags("desktop");
+
+test("logs are saved and garbage-collected after TTL", async () => {
+    indexedDB.deleteDatabase("test_db");
+    const logger = new Logger("test_db");
+    await logger.log("foo");
+    await logger.log("bar");
+    expect(await logger.getLogs()).toEqual(["foo", "bar"]);
+    await advanceTime(Logger.LOG_TTL + 1000);
+    expect(await logger.getLogs()).toEqual([]);
+    indexedDB.deleteDatabase("test_db");
+});
```

Comment:
Can use `beforeEach` and `afterEach` for `deleteDatabase()` on the suite, to make sure this is properly delete when running many tests while this one failed in the middle of it.